client/src/js/statsTable.js

```diff
@@ -1,108 +1,44 @@
-let responseData = {data: [['teams', 'wins ⬇️', 'losses ⬇️', 'ties ⬇️', 'championships ⬇️'], ['bombers', 20, 0, 3, 18], ['jets', 40, 7, 2, 3], ['cosmonauts', 10, 100, 86, 8], ['korichugh', 90, 3, 56, 6], ['chiliPeppers', 28, 37, 2, 0]]};
+let responseData = {data: [['teams', 'wins', 'losses', 'ties', 'championships'], ['bombers', 20, 0, 3, 18], ['jets', 40, 7, 2, 3], ['cosmonauts', 10, 100, 86, 8], ['korichugh', 90, 3, 56, 6], ['chiliPeppers', 28, 37, 2, 0]]};
 let currentSortingColumn = 1;
 let isAscending = false;
-let sortedData;
 
-sortedData = sortData();
-createAndAddTable(sortedData);
+var sortedData = sortData();
+updateTable(sortedData);
 
-function sortData(col = 1, isAscending = false) {
-  if (col === 0 || col >= responseData.data[0].length) {
-    console.log('bad column. aborting sortData()!');
-    return;
-  }
-  
-  let runOnce = false;
-  var sortedData = responseData.data.slice(1).sort((a, b) => {
-    
-    if (isAscending) {
-      if (!runOnce) console.log('sortData() sorting by ascending');
-      runOnce = true;
+function updateTable(data) {
+  var html = nunjucks.render('statsTable.html', {
+    table: data,
+    currentSortingColumn: currentSortingColumn,
+    isAscending: isAscending
+  });
+  var table = document.querySelector('#statsTable');
+  table.innerHTML = html;
+}
 
-      return a[col] - b[col];
-    }
-    else {
-      if (!runOnce) console.log('sortData() sorting by descending');
-      runOnce = true;
-      
-      return b[col] - a[col];
+function sortData() {
```

Comment:
`currentSortingColumn` and `isAscending` are globally declared anyway, so don't require them to be passed.